pkg/volume/util/operationexecutor/operation_executor_test.go

```diff
@@ -843,6 +843,10 @@ func setup() (chan interface{}, chan interface{}, OperationExecutor) {
 // This function starts by writing to ch and blocks on the quit channel
 // until it is closed by the currently running test
 func startOperationAndBlock(ch chan<- interface{}, quit <-chan interface{}) {
-	ch <- nil
+	select {
+	case ch <- nil:
+	case <-time.After(10 * time.Second):
+		klog.Warning("Possible race condition: started operation not checked in test")
+	}
```

Comment:
/hold

IIUIC if the timeout expires the test may pass when it should fail, in this case if the channel is blocked and there is a goroutine leak the failure seems legit